src/event.rs

```diff
@@ -467,6 +467,21 @@ pub enum DeviceEvent {
     ///
     /// This represents raw, unfiltered physical motion. Not to be confused with
     /// [`WindowEvent::CursorMoved`].
+    ///
+    /// ## Platform-specific
+    ///
+    /// **Web:** Always returns OS accelerated data unless [`CursorGrabMode::Confined`] is used depending on browser support, see
```

Comment:
locked, and delta is unaccelerated.

Reply:
It says "accelerated data **unless**".
I removed the negation which makes it easier to understand though.

Thanks!